### PR TITLE
Implement check media async API

### PR DESCRIPTION
**Pull request by tssujt**

### CODE REVIEW

File: wechatpy/client/api/wxa.py [hunk 1]
```diff
@@ -519,6 +519,23 @@ def check_text_security(self, content):
             }
         )
 
+    def check_media_async(self, media_url: str, media_type: int):
```
Comment (messense) [inline wechatpy/client/api/wxa.py]: This type hint won't work on Python 2.7

Comment (tssujt) [reply wechatpy/client/api/wxa.py]: > This type hint won't work on Python 2.7

@messense Fixed

**Comment by messense** [comment]
CleanShot 2021-03-16 at 23 28 15@2x

这个文档怎么看起来像是 JSON 而不是 XML?

**Comment by tssujt** [comment]
> CleanShot 2021-03-16 at 23 28 15@2x
> 
> 这个文档怎么看起来像是 JSON 而不是 XML?

image

**Comment by messense** [comment]
现在 `parse_message` 只支持 XML 格式：
https://github.com/wechatpy/wechatpy/blob/635271ac1b91b4beeab68441e3d83eed2d27a454/wechatpy/parser.py#L17-L27

**Comment by tssujt** [comment]
> 现在 `parse_message` 只支持 XML 格式：
> 
> https://github.com/wechatpy/wechatpy/blob/635271ac1b91b4beeab68441e3d83eed2d27a454/wechatpy/parser.py#L17-L27

I will support parsing JSON later.

**Comment by tssujt** [comment]
@messense I found this kind of message also support XML. Ref: https://github.com/silenceper/wechat/blob/813684e55535af7fb4cb29c3e3a9e2a02384b34e/officialaccount/message/message.go#L159

**Comment by messense** [comment]
> @messense I found this kind of message also support XML. Ref: https://github.com/silenceper/wechat/blob/813684e55535af7fb4cb29c3e3a9e2a02384b34e/officialaccount/message/message.go#L159

不知道有没有人测试过。。。

**Comment by messense** [comment]
先合并了，有人遇到问题再改😅

**Pull request merged**